source/features/forked-to.tsx

```diff
@@ -47,22 +46,29 @@ async function updateUI(forks: string[]): Promise<void> {
 	}
 
 	document.body.classList.add('rgh-forked-to');
-	const forkCounter = await elementReady('.social-count[href$="/network/members"]', {waitForChildren: false});
+	const forkContainer = select('.pagehead-actions > li > div.float-left')!;
```

Comment:
This is a bit less brittle:

```suggestion
	const forkContainer = select('.pagehead-actions .octicon-repo-forked')!.closest('.float-left')!;
```

(Untested but it should work.)